test/Store.ml

```diff
@@ -0,0 +1,27 @@
+include EsyLib.Store
```

Comment:
It probably would make sense for this test to be in reason, since EsyLib.Store is in reason. However, when I tried to create a reason test file, like: 
__`MySimpleTest.re`__:
```
let%test "hello" = false;
```

I'd get this error:
```
jbuilder runtest test
esy-bash: executing bash command: jbuilder runtest test | nonce: 1
 -- script file: C:/Users/bryph/AppData/Local/Temp/__esy-bash__1531246373585__1__.sh
         ppx test/Store2.re.pp.ml (exit 1)
(cd _build/default && ./.ppx/ppx_inline_test+ppxlib.runner/ppx.exe --dump-ast --cookie "library-name=\"Test\"" -diff-cmd - -o test/Store2.re.pp.ml --impl test/Store2.re.ml)
File "_none_", line 1:
Error: Attributes not allowed here
```

I think maybe there is some interaction with the `ppx_inline_test` and reason, or perhaps something simple I missed in setting it up? LMK if you have any ideas @andreypopp and I'm happy to switch the test over